tests/Avalonia.Markup.UnitTests/Data/BindingTests_Converters.cs

```diff
@@ -52,6 +51,28 @@ public void StringFormatConverter_Should_Be_Used_When_Binding_Has_StringFormat()
             }
         }
 
+        public class When_Binding_To_Decimal
+        {
+            [Fact]
+            public void When_Binding_Has_StringFormat_Without_Braces()
+            {
+                var textBlock = new TextBlock
+                {
+                    DataContext = new Class1(),
+                };
+
+                var binding = new Binding(nameof(Class1.DecimalValue))
+                {
+                    StringFormat = "c2",
+                };
```

Comment:
It's a reflection bindings test. 
If you want to test compiled bindings, would be easier to add XAML tests to the Avalonia.Markup.Xaml.UnitTests project.

Reply:
I've also seen that ReflectionBing doesn't work. I'll add it tomorrow